packages/rtop/test/ops_lib/CMakeLists.txt

```diff
@@ -38,6 +38,7 @@ TRIBITS_ADD_EXECUTABLE_AND_TEST(
     TOpEleWiseDivide_UnitTests.cpp
     TOpEleWiseProdUpdate_UnitTests.cpp
     TOpEleWiseProd_UnitTests.cpp
+    TOpPairWiseMax_UnitTests.cpp
```

Comment:
@bartlettroscoe I wrote and added a unittest in rtop but I'm not exactly sure if it is running. I don't see the test when I grep for it in the test log.

Reply:
You just build the test and run it.  I pulled the branch, built the test executable, and ran it as:

```
./RTOp_opsUnitTests.exe --group=TOpPairWiseMax* --details=ALL --verbose
Teuchos::GlobalMPISession::GlobalMPISession(): started processor with name crf450.srn.sandia.gov and rank 0!

***
*** Unit test suite ...
***


Sorting tests by group name then by the order they were added ... (time = 7.8e-05)

Running unit tests ...


78. TOpPairWiseMax_double_unitStride_UnitTest ... 
 alpha = 2
 v0 = {0.68, -0.211, 0.566, 0.597}
 v1 = {0.823, -0.605, -0.33, 0.536}
 orig_z0 = {-0.444, 0.108, -0.0452, 0.258}
 z0 = {1.65, -0.422, 1.13, 1.19}
 expected_z0 = {1.65, -0.422, 1.13, 1.19}
 Comparing constSubVectorViewAsArray(z0) == constSubVectorViewAsArray(expected_z0) ... passed
 [Passed] (0.000117 sec)

79. TOpPairWiseMax_double_nonunitStride_UnitTest ... 
 alpha = 2
 v0 = {-0.27, 0.0268, 0.904, 0.832}
 v1 = {0.271, 0.435, -0.717, 0.214}
 orig_z0 = {-0.967, -0.514, -0.726, 0.608}
 z0 = {0.543, 0.869, 1.81, 1.66}
 expected_z0 = {0.543, 0.869, 1.81, 1.66}
 Comparing constSubVectorViewAsArray(z0) == constSubVectorViewAsArray(expected_z0) ... passed
 [Passed] (4.6e-05 sec)

Total Time: 0.000412 sec

Summary: total = 97, run = 2, passed = 2, failed = 0

End Result: TEST PASSED
```